src/module/settings.js

```diff
@@ -162,6 +162,15 @@ export function registerSystemSettings() {
     },
     default: CONFIG.YZE_COMBAT.ultimateMaxDrawSize,
   });
+
+  game.settings.register(MODULE_ID, SETTINGS_KEYS.CONFIGURED, {
+    name: 'SETTINGS.Configured',
+    hint: 'SETTINGS.ConfiguredHint',
+    scope: 'world',
+    config: false,
```

Comment:
```suggestion
    config: true,
```
Proposing to display this setting so the GM can reset the configuration.